### DISCUSSION TITLE
Compiling Tasmota V12.1.1 with USE_RF_SENSOR, the RFSend is NOT present!

### DISCUSSION THREAD

**Discussion by lumanga** [arendst/Tasmota#16689] [Q&A] [answered]
### PROBLEM DESCRIPTION
Dear all,
I have compilet the Tasmota, with Visual Studio Core, starting from "standard base" tasmota V12.1.1 adding the 
 #define USE_RF_SENSOR string.

Flashing the firmare on my device, on the web page of the Wemos D1 Mini, in the "Configure Module" menu, appear ONLY the "RF Sensor" option, **BUT I NEED the  RF Send option.**

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [x] Self-compiled
- [x] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by sfromis** [comment, marked as answer]
Don't open an "issue" which is for reason to suspect Tasmota bugs, when you just did not find out how to make it work.

AFAIK, you must have `#define USE_RC_SWITCH` along with `#define USE_RF_SENSOR`. Both should already be set in tasmota-sensors, meaning that you need no self-compile if flashing that precompiled binary.
